stream_alert/alert_processor/outputs.py

```diff
@@ -581,12 +581,30 @@ def dispatch(self, **kwargs):
         alert = kwargs['alert']
         alert_string = json.dumps(alert['record'])
         function_name = self.config[self.__service__][kwargs['descriptor']]
+        parts = function_name.split(':')
+
+        # Check to see if there is an optional qualifier included here
+        if len(parts) == 2 or len(parts) == 8:
```

Comment:
Why are the two expected lengths 2 and 8? Can you provide examples of these two cases in the comments?